tests/system/test_session_api.py

```diff
@@ -346,6 +355,11 @@ def _make_attributes(db_instance, **kwargs):
         "net.host.name": "spanner.googleapis.com",
         "db.instance": db_instance,
     }
+
+    if ot_helpers.HAS_OPENTELEMETRY_INSTALLED:
```

Comment:
@odeke-em This will add a lot of code duplication to add these 2 attributes since we need them in all the test files. Earlier I was assuming we need it only in one file.
May be we should move this code to _helpers.py file and refer these attributes from there similar to your previous approach. This will remove all these duplicates and these 2 attributes can be directly added to base attributes field without additional conditions.

Reply:
Yeah it was indeed cumbersome to duplicate that code. Sure let me revert it back to what I had.

Reply:
Refactor done @harshachinta